lib/index.js

```diff
@@ -0,0 +1,2 @@
+// Do not allow to import this module
+throw new Error("Module 'metad' not allowed to import");
```

Comment:
this will not allow other devs to import this module unexpectedly or cause their program to have bugs because of this package 😄